docs/audit-log-filter-variables.md

```diff
@@ -0,0 +1,728 @@
+
+* `user_name` - a selected user name in either the `user_name`@`host_name` format or `%`.
+
+* `filter_name` - a selected filter name as a string.
+
+#### Returns
+
+This function returns either an `OK` for success or an error message for failure.
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_filter_set_user('user-name@localhost', 'filter-name');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------------------------------------------+
+    | audit_log_filter_set_user('user-name@localhost', 'filter-name')  |
+    +-------------------------------------------------------------------+
+    | OK                                                                |
+    +-------------------------------------------------------------------+
+    ```
+
+## Audit log filter options and variables
+
+| Name |
+| --- |
+| [`audit-log-filter`](#audit-logvalue) |
+| [`audit_log_buffer_size`](#audit_log_buffer_size) |
+| [`audit_log_filter_compression`](#audit_log_filter_filter_compression) |
+| [`audit_log_filter_database`](#audit_log_filter_database) |
+| [`audit_log_filter_disable`](#audit_log_filter_disable) |
+| [`audit_log_filter_encryption`](#audit_log_filter_encryption) |
+| [`audit_log_filter_file`](#audit_log_filter_file) |
+| [`audit_log_filter_format`](#audit_log_filter_format) |
+| [`audit_log_filter_format_unix_timestamp`](#audit_log_filter_format_unix_timestamp) |
+| [`audit_log_filter_handler`](#audit_log_filter_handler) |
+| [`audit_log_filter_key_derivation_iterations_count_mean`](#audit_log_filter_key_derivation_iterations_count_mean) |
+| [`audit_log_filter_max_size`](#audit_log_filter_max_size) |
+| [`audit_log_filter_keep_password_history_keep_days`](#audit_log_filter_keep_password_history_keep_days) |
+| [`audit_log_filter_prune_seconds`](#audit_log_filter_prune_seconds) |
+| [`audit_log_filter_read_buffer_size`](#audit_log_filter_read_buffer_size) |
+| [`audit_log_filter_rotate_on_size`](#audit_log_filter_rotate_on_size) |
+| [`audit_log_filter_strategy`](#audit_log_filter_strategy) |
+| [`audit_log_filter_syslog_tag`](#audit_log_filter_syslog_tag) |
+| [`audit_log_filter_syslog_priority`](#audit_log_filter_syslog_priority) |
+
+
+
+### `audit-log-filter`
+
+| Option       | Description                     |
+|--------------|---------------------------------|
+| Command-line | --audit-log-filter[=value]|
+| Dynamic      | No                              |
+| Scope        |                                 |
+| Data type    | Enumeration                     |
+| Default      | ON                              |
+
+This option determines how, at startup, the server loads the `audit_log_filter` component. The component must be registered. 
+
+The valid values are the following:
+
+* ON
+* OFF
+* FORCE
+* FORCE_PLUS_PERMANENT
+
+
+### `audit_log_filter_buffer_size`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-buffer-size |
```

Comment:
Names in this format should be changed as well --audit-log-filter-buffer-size -> --audit-log-filter.buffer-size

Reply:
fixed